crates/node/src/service/rpc/metrics.rs

```diff
@@ -113,15 +113,15 @@ impl RpcMetrics {
         tracing::trace!(target: "rpc_metrics", "[{transport_label}] on_response started_at={:?}", now);
         tracing::trace!(target: "rpc_metrics::extra", "[{transport_label}] result={}", rp.as_result());
 
-        let micros = now.elapsed().as_micros();
+        let millis = now.elapsed().as_millis();
         tracing::debug!(
             target: "rpc_metrics",
-            "[{transport_label}] {} call took {} μs",
+            "[{transport_label}] {} call took {} ms",
```

Comment:
can you use Duration {:?} (Debug) impl instead? it'll add the s/ms/us suffix depending on the value (i think that's what we used to do before you switched the codebase to tracing

Reply:
Good catch !
Added